client/blocks/reader-full-post/style.scss

```diff
@@ -131,15 +131,6 @@ $reader-full-post-desktop-min: $reader-full-post-sidebar-width + $reader-full-po
 		font-size: $font-body;
 		line-height: 24px;
 	}
-
-	.comment-button__label {
```

Comment:
After working with the line height and making sure font sizes and such were consistent, flexbox took care of the alignment so these manual adjustments were unnecessary.